### DISCUSSION TITLE
How to avoid ban from TG because of too many forwards?

### DISCUSSION THREAD

**Discussion by tissole** [aahnik/tgcf#90] [Q&A] [answered]
Now that the script can run on the cloud basically non-stop there's a risk of TG ban if a user copies too many files? I heard that were some cases, but nothing too well documented. How many files are you copying guys and do you encounter any ban? It is temporary or permanent? Are there any references in the TG official documentation of any limits?

**Comment by aahnik** [comment, marked as answer]
I don't know about this.  Generally, telegram will not ban you for sending messages. **But if you spam others, and if others report your account then telegram may ban you.**

Telegram may suspend your account temporarily for **unusual activity.**

Use `tgcf` at your own risk. It is distributed under MIT License.

![image](https://user-images.githubusercontent.com/66209958/115960013-bba18180-a52c-11eb-9257-ee3c3bad217c.png)

>  but nothing too well documented

that is true.

Use a bot account for better safety. To generate API ID and API HASH use a different phone no, in [my.telegram.org](https://my.telegram.org)

**Comment by tissole** [comment]
I never spammed anyone, I use this for personal backup of files in my private channel where I'm the only member. But I heard that some people think that TG might consider forwarding spamming. I'm skeptical of this. Others say that this is some sort of scraping, not sure about that, that's why I'm asking.

From what I understand the bot can only be used for copying between owner chats, in the case of files from other channels a personal account with a phone number is required. 

"Telegram may suspend your account temporarily for unusual activity."
You're right. A few months ago I tested an old TG client and got an instant ban. I wrote to TG and explain and they lifted the ban in a few hours.
